config/samples/modeljobs_v1alpha1_savedmodel_extract.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: modeljobs.caicloud.io/v1alpha1
+kind: ModelJob
+metadata:
+  name: modeljob-savedmodel-extract
+  namespace: kube-system
+spec:
+  # Add fields here
+  type: "Extract"
```

Comment:
If we have `extraction` field, why do we need the type here?

Reply:
the type is designed, it should be deleted. I delelte it now.